images/java/src/main/java/io/iron/lambda/Launcher.java

```diff
@@ -131,6 +131,8 @@ private void runMethod(Method lambdaMethod, String payload, Class cls, String ha
         if (!processed) {
             System.err.println(String.format("Handler %s with simple, POJO, or IO(input/output) types not found", handlerName));
         }
+        String jsonInString = ClassTypeHelper.gson.toJson(result);
```

Comment:
Not 100% sure about this lib's behavior. Is it going to throw an exception or something ifthe result is not serializable? Do you want to just wrap a try catch around it?